action_transaction_test.go

```diff
@@ -187,6 +187,30 @@ func Test_RevertTransaction(t *testing.T) {
 	})
 }
 
+func Test_RecordTransaction(t *testing.T) {
+	//var secondTransaction *Transaction
+	ctx, client, _ := initSimpleTestCase(t)
```

Comment:
Please rewrite test with `t.Run("test case name", (...)` or rename test function and give it meaningful name.

Also please write test code in `// given //when // then` form

Reply:
Added